handlers/device.go

```diff
@@ -143,3 +144,80 @@ func (h *DeviceHandler) GetDeviceByName(w http.ResponseWriter, r *http.Request)
 
 	return nil
 }
+
+// Handle API endpoint for getting device measurements
```

Comment:
I see you did not implement the spec for this endpoint yet:
- query params
- response you described

So I am guessing you wanted feedback up until this point.

Reply:
I've added the API docs in [2a1318e](https://github.com/energietransitie/twomes-backoffice-api/pull/106/commits/2a1318e4532eba4aafa850042b19e3ff70d2e144), please let me know if this is sufficient.

Reply:
The openAPI spec addition looks great, thanks!

That's not what I meant however, but great that you did it. But I meant that the implementation does not have all the functionality of your spec yet. So my assumption is that you are planning on doing this, but already wanted some input from me up until this point. Am I correct?

Reply:
Yes, we will still need to add the filters for `property`, `start` and `end`. The optional `granularity` parameter will probably be added in another PR given the complexity.

Reply:
`property`, `start` and `end` filters have been added.